core/server/src/main/java/alluxio/master/file/meta/InodePathPair.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.file.meta;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * This class represents a pair of {@link InodePath}s.
+ */
+@ThreadSafe
+public class InodePathPair implements AutoCloseable {
```

Comment:
I made it final. However, should I also remove `final` from `alluxio.collections.Pair` to extend it?